chsdi/templates/htmlpopup/bikesharing.mako

```diff
@@ -0,0 +1,20 @@
+<%inherit file="base.mako"/>
+
+
+<%def name="table_body(c, lang)">
+<%
+	lang = lang if lang != 'rm' else 'de'
+	time = 'time' + lang
+	rent = 'rent' + lang
+	link = 'link' + lang
+	description = 'description' + lang
+	returN = 'return' + lang
```

Comment:
Instead of creating a variable her, you could do the key creation directly on access. e.g.

`c['attributes']['time' + lang]`